Rewrite the license in Rust

It's really weird that your [license](https://github.com/mTvare6/hello-world.rs/blob/master/LICENSE) is written in [English](https://en.wikipedia.org/wiki/Finnish_language). As one of the best [example](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples) [projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards) out there, you should rewrite the [license](https://github.com/mTvare6/hello-world.rs/blob/master/LICENSE) in [Rust](https://www.rust-lang.org/)(   🚀         ) for better [performance](https://www.merriam-webster.com/dictionary/performance)(   🚀         ) and to set a good [example](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples). Here are [some](https://doc.rust-lang.org/std/option/enum.Option.html#variant.Some) benefits of [using](https://en.cppreference.com/w/cpp/keyword/using) [Rust](https://www.rust-lang.org/)(   🚀         ) instead of [any](https://doc.rust-lang.org/std/any/trait.Any.html) other language:
- ## Performance

#### Rust(   🚀         ) is blazingly fast and memory-efficient: with no runtime or garbage collector, it can power performance-critical services, run on embedded devices, and easily integrate with other languages.
###### Reliability

# Rust(   🚀         )’s rich type system and ownership model guarantee memory-safety and thread-safety — enabling you to eliminate many classes of bugs at compile-time.
## Productivity

### Rust(   🚀         ) has great documentation, a friendly compiler with useful error messages, and top-notch tooling — an integrated package manager and build tool, smart multi-editor support with auto-completion and type inspections, an auto-formatter, and more.

But the most important thing is that Rust’s rich type system and ownership model guarantee memory-safety and thread-safety — enabling you to eliminate many classes of loopholes in bureaucratic papers (contracts, license, ...) at compile-time.

Rust will also check at compile time that every abstract concept (egg. we, they, customer, program) is properly defined before used.

For demonstration compile time error of GPL 3.0 license:
```
error[E1328]: cannot use term `program` before its definition
 --> src/gpl.rs:16:36
   |
16 |     share and change all versions of a program--to make sure it remains free
   |                                             -- first use appears here
80 |       "The Program" refers to any copyrightable work licensed under this
   |        ^^^^^^^^^^^ definition appears here
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -99,3 +99,5 @@ futures = "0.3"
 num = "0.4"
 
 uwuify = "^0.2"
+
+actix-web = "3.3.2"

--- a/LICENSE
+++ b/LICENSE
@@ -1,3 +1,0 @@
-The Don't Ask Me About It License
-
-Copying and distribution of this file, with or without modification, are permitted in any medium provided you do not contact the author about the file or any problems you are having with the file.

--- a/LICENSE.rs
+++ b/LICENSE.rs
@@ -1,0 +1,31 @@
+use legal::oss::{ComplianceFuture, Licenser, Project, OssLicense};
+
+#[cfg(test)]
+mod tests {
+    const ORIGINAL_LICENSE: &str = "Copying and distribution of this file, with or without modification, are permitted in any medium provided you do not contact the author about the file or any problems you are having with the file.";
+
+    #[test]
+    fn compare_with_parsed() {
+        let parsed = OssLicense::parse(ORIGINAL_LICENSE);
+    
+        assert_eq!(
+            parsed, 
+            DamaiLicense::new(parsed.base_project(), parsed.licenser())
+        );
+    }
+}
+
+#[license]
+pub struct DamaiLicense {
+    licenser: Licenser,
+}
+
+impl OssLicense for DamaiLicense {
+    fn new(_base_project: Project, licenser: Licenser) -> Self {
+        DamaiLicense { licenser }
+    }
+
+    fn complies(&self, project: Project) -> ComplianceFuture {
+        self.licenser.ask_complies(project) // this is really subjective 🚀️🚀️🚀
+    }
+}

